spec.bs

```diff
@@ -1024,41 +1025,101 @@ partial interface Navigator {
 };
 </xmp>
 
-These updates have 2 triggers: the {{Navigator/updateAdInterestGroups()}} API, which updates all interest
-
-groups owned by the origin of the current frame, and post-auction updates, which, at the conclusion
-of {{Window/navigator}}.{{Navigator/runAdAuction()}}, updates the interest groups of all bidder origins that participated in the
-
-
-auction.
+The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
 
-<div algorithm="interest group update">
-
-Both update methods trigger the same algorithm, <dfn>interest group update</dfn> which is described below, and takes a list of owners to
-update called "owners":
+1. [=in parallel=] execute the algorithm [=interest group update=] using « [=this=]'s
+  [=relevant settings object=]'s [=environment/top-level origin=] » as `owners`.
+  
+<div algorithm>
+  To <dfn>interest group update</dfn> a given [=list=] of [=url/origins=] |owners|:

```

Comment:
The english "To interest group update a given" kinda of feels awkward. Feel free to write something like `<dfn lt="interest group update">update an interest group</dfn>` where `lt` is "link text" that you use elsewhere to link to this algorithm. It's often used to make the in-context english of dfn titles more comprehensible.